src/domains/sign/network.tf

```diff
@@ -223,3 +256,24 @@ resource "azurerm_private_endpoint" "io_sign_issuer_func" {
 
   tags = var.tags
 }
+
+resource "azurerm_private_endpoint" "io_sign_support_func" {
```

Comment:
@manuelcoppotelli I remember that we need a private endpoint for staging slot, right?
cc @lucacavallaro

Reply:
Correct, @pasqualedevita.
We have already planned it